static/styles/rewards/claim-table.css

```diff
@@ -313,9 +313,6 @@ thead {
   overflow: hidden;
   /* box-shadow: 0 12px 64px #00000010; */
 }
-table * {
-  text-wrap: nowrap;
-}
```

Comment:
That is there for a reason. Have you seen how it breaks the page on mobile?

I estimated the task to be two hours, because it requires restructuring the table html.